app/setup/scss/mixins/_half-border.scss

```diff
@@ -0,0 +1,42 @@
+////
+// Adds half-border to the element
+//
+// @param  {String}   Color of the border
+//
+////
+
+@mixin half-border($border-color: $color-grayscale--500) {
+  @if $border-color != null {
```

Comment:
- [ ] Please remove condition. Not needed anymore, since you added a fallback.

Reply:
done